### DISCUSSION TITLE
New ExplorerPatcher taskbar implementation solved mouse stutter!

### DISCUSSION THREAD

**Discussion by GTR8000** [valinet/ExplorerPatcher#3532] [General]
I'm happy to report that the new Windows 10 (ExplorerPatcher) taskbar has solved my stuttering mouse issue.

I upgraded from Win 10 last night and after installing ExplorerPatch 22621.3527.65.5 and setting the taskbar style to Windows 10, I started having very noticeable stuttering of mouse movements whenever I would open/close windows along with a few other tasks. CPU usage never spiked above 10%, so it seemed odd. Reverted back to Windows 11 style taskbar and stuttering went away. Hmmm. I then installed the latest pre-release of ExplorerPatch, tried Windows 10 style, same stuttering. I then tried the new Windows 10 (ExplorerPatcher) style, and the stuttering magically disappeared.

My hunch is that the mouse stuttering is an issue that affects my particular combination of motherboard/CPU, not an issue that affects all users. If that were the case, surely others would be complaining about the stuttering when on the most recent versions of Win 11 using the standard Windows 10 taskbar style. I'm just happy to have this issue solved, and look forward to the further development of the new taskbar implementation (I'm a little bummed that 7+ Taskbar Tweaker no longer works)

Win 11 Pro 23H2 (22631.3958)
ExplorerPatch 22621.3880.66.4
MSI PRO B550M-VC WIFI
AMD Ryzen 5 5600G

**Comment by Amrsatrio** [comment]
Yeah I get you, after all I'm getting pressured to bring 7+TT features into ep_taskbar anyways...

**Reply by GTR8000** [reply]
After pulling my hair out trying to figure out why the mouse was stuttering so badly, and seriously considering rolling back to Win 10, losing the 7+ TT features is a small price to pay for the new taskbar fixing the issue. Thanks!

The one thing I am sad to lose, which I believe is a feature exclusive to 7+ TT (unless I'm missing the options elsewhere) is to group the taskbar buttons by application ID and to not combine those grouped buttons. Basically I hate seeing the labels on the taskbar, only want to see the icons, but it's annoying when multiple windows of the same application are grouped together.

**Reply by pyrates999** [reply]
7+ Taskbar Tweaker is compatible with windows 11 using the windows 10 feature in EP: https://ramensoftware.com/7-taskbar-tweaker-on-windows-11-with-windows-10s-taskbar

**Reply by Amrsatrio** [reply]
@pyrates999 You're getting it wrong. OP said that ep_taskbar solves their mouse problems, however it is not compatible with 7+TT. If OP uses the stock 10 taskbar, the mouse problems will occur but 7+TT will be usable.

**Reply by pyrates999** [reply]
Ohhhhhhh sorry.  I thought the url I posted for 7+ Taskbar Tweaker indicated it worked with EP's implementation of the windows 10 taskbar in windows 11.

**Reply by GTR8000** [reply]
@pyrates999 There is a new Windows 10 style taskbar for ExplorerPatch called **Windows 10 (ExplorerPatcher)**, not to be confused with the original implementation of the Windows 10 style taskbar. It is available in the latest pre-releases. More info here about it:

https://github.com/valinet/ExplorerPatcher/wiki/ExplorerPatcher's-taskbar-implementation

**Reply by pyrates999** [reply]
That's what I was referring to.  Thanks for the info so others know what it is.
